projects/plugins/jetpack/changelog/spread-payments-intro

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: enhancement
+
+Payments Block: Use the payments introduction whenever selecting an eligible payments block - payments button and donations
```

Comment:
Since this PR doesn't target the master branch, do we actually need this changelogger? Was it added to pass the pre-commit hooks?

Reply:
It was added to make the pre-commit hook pass, but also I made this branch target the master one because otherwise this PR would cover all of `try/unified-payment-intro` which is unnecessary. Depending upon when `try/unified-payment-intro` ships, this branch might get changed to target master.